src/content/docs/guides/JSON/0-json_intro.mdx

```diff
@@ -191,7 +230,45 @@ else
 
   FreeJson(gameData);
 ```
+</TabItem>
+<TabItem label="Object-Oriented">
+
+```csharp
+using SplashKitSDK;
+
+namespace GameDataManagement
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Define the file path
```

Comment:
Similar in this one, the comments look out of place when comparing to the C++ and C# top level statement versions.